CMakeSettings.json

```diff
@@ -3,7 +3,7 @@
     {
       "buildCommandArgs": "-v",
       "buildRoot": "${projectDir}\\out\\build\\${name}",
-      "cmakeCommandArgs": "-DCMAKE_TOOLCHAIN_FILE=C:\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake",
+      "cmakeCommandArgs": "-DCMAKE_TOOLCHAIN_FILE=C:\\Users\\Tim\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake",
```

Comment:
I think the `CMakeSettings.json` shouldn't be tracked in git. Consider removing it since you're hard-coding your user file path which will only be valid for you.

Reply:
I noticed this last night after I was checking the Pr myself. I was afraid to straight up delete the file because I thought it might delete it locally. I’m not too familiar with git, so it would nice if you could tell me how I should resolve this. Thanks!

Reply:
Yes, you can remove the file from future tracking, e.g. via this answer: https://stackoverflow.com/a/1274447
However, this will not remove the file from the history.

You could retroactively delete it from the history (preferred): https://stackoverflow.com/a/64563565